app/ports/management/commands/update-portinfo.py

```diff
@@ -0,0 +1,50 @@
+from django.core.management.base import BaseCommand, CommandError
```

Comment:
https://github.com/macports/macports-webapp/pull/33#discussion_r302763269 I have used the name `update-portinfo`.